lib/web_ui/lib/src/engine/html/shaders/shader.dart

```diff
@@ -22,6 +22,21 @@ import 'vertex_shaders.dart';
 const double kFltEpsilon = 1.19209290E-07; // == 1 / (2 ^ 23)
 const double kFltEpsilonSquared = 1.19209290E-07 * 1.19209290E-07;
 
+class TempCanvas {
```

Comment:
nit: is this really a "temp" canvas? It feel more like a "permanent" canvas :) Maybe call it "SharedCanvas"?